code/game/objects/items/mail.dm

```diff
@@ -332,3 +352,164 @@
 /obj/item/paper/fluff/junkmail_generic/Initialize(mapload)
 	default_raw_text = pick(GLOB.junkmail_messages)
 	return ..()
+
+/obj/item/mail/traitor
+	var/armed = FALSE
+	var/datum/weakref/made_by_ref
+	// If somehow mind will disappear, admins will still have info about creator
```

Comment:
```suggestion
	/// Cached information about who made it for logging purposes
```

You're not using the mind at any point in logging code, even when it still exists

Reply:
Got it